docs/install/lotus/troubleshooting.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Troubleshooting Lotus installation
+description: If you're having issues install Lotus on your system, try running through some of these troubleshooting steps.
+---
+
+# Troubleshooting
+
+If you're having trouble installing Lotus, try running through some of these troubleshooting steps. If you're still having trouble, feel free to reach out to the team on the `#fil-help` channel in the Filecoin Slack.
+
+## Update your system
+
+This sounds simple, but a common reason users aren't able to run Filecoin is because their system isn't up to date.
```

Comment:
Does this mean operating system? hardware system?

Reply:
Operating system.